src/components/Layout/Footer/index.js

```diff
@@ -0,0 +1,3 @@
+import Footer from './Footer';
+
+export default Footer;
```

Comment:
By the way, this could be more concise as:

```suggestion
export { default } from './Footer';
```

You can see us using this pattern in a number of places in askdarcel-web, such as https://github.com/ShelterTechSF/askdarcel-web/blob/016ba8253cc6d4210f5022dbb774a06e024a6c0e/app/pages/HomePage/components/SearchBar/index.js#L1.